explorer/app/components/Layout/components/Footer/footer.tsx

```diff
@@ -40,7 +43,12 @@ export const Footer = ({ footer }: Props): JSX.Element => {
             </NLink>
           ))}
           {feedbackForm && <>{/* TODO feedback form */}</>}
-          <Socials buttonSize="small" socials={socials} sx={{ gap: 2 }} />
+          <Socials
+            buttonSize="small"
+            IconButtonElType={IconButtonSocialsFooter}
```

Comment:
`Socials` now takes a styled IconButton.

Reply:
Can you give me some background on why this was necessary with respect to the palette updates?

Reply:
IconButton prop colour expects a top level palette colour like "primary" or "smoke" or "success" or "info". It then takes the defined `main` value for the palette e.g. `success.main`. The social icons are coloured "smoke.light" and so we are unable to specify this colour (using `smoke` would return `smoke.main`). Unfortunately, IconButton does not have the `variant` prop (with which I could then create a special variant and modify the colour in theme). Creating a styled button was the next best "workaround" in this particular instance.